src/core/Akka.Remote/Endpoint.cs

```diff
@@ -102,73 +102,74 @@ public DefaultMessageDispatcher(ExtendedActorSystem system, IRemoteActorRefProvi
                 }
             }
 
-            //message is intended for a local recipient
-            else if (recipient is ILocalRef or RepointableActorRef && recipient.IsLocal)
+            else switch (recipient)
```

Comment:
ReSharper suggestion to turn these into `switch` statements rather than `if` statements - zero change in the logic.